### ISSUE TITLE
collectd snmp interval by Collect

### ISSUE THREAD

**Issue by Xyas** [collectd/collectd#1494]
Hi Guys,

Im trying to collect data from one host through snmp plugin, but i need to collect different data on different intervals. I cannot duplicate host entry because an error is generated, and adding a new interval after a new 'Collect' entry on Host, wont generate any kind of error, but its ignored.

The reason of this es because i want to collect some values only once a day, and other values every 5 minutos, an so on..., same host, different intervalos Per Data block.

Thanks.

**Comment by melankh** [comment]
I agree with Xyas. I've the same problem. 
Some Brocade 32 bits counters turns too fast. I have to poll them every 15 sec. But I don't want to overload my switch with checking every others values at the same rate.

**Comment by sbarrier** [comment]
I agree with the previous comments such feature would be nice for equipment like router where you need to collect multiple snmp value at different interval to avoid such issue where the snmp plugin fail :

2016-09-29T01:32:00-07:00 X collectd[21255]: snmp plugin: host mlx: snmp_sess_synch_response failed: Timeout
2016-09-29T01:32:00-07:00 X collectd[21255]: read-function of plugin `snmp-mlx' failed. Will suspend it for 40.000 seconds.

**Comment by crab86** [comment]
It would be really appreciated if someone could implement a configurable timeout for snmp plugin. We do have several routers to query for interface statistics and currently we do get "snmp_sess_synch_response failed: Timeout" message. We also do get a timeout if we do use snmpwalk for the same devices. But using snmpwalk with a timeout of 10s (smpwalk -t 10) we do get the results we need.

**Comment by toni-moreno** [comment]
Hi @Xyas @crab86 , if you would like to test other tools, We have built a  [snmp collecting tool](https://github.com/toni-moreno/snmpcollector/wiki)  to get data to put into influxdb database, and you can configure any snmp related configuration per device. You have also a nice web user interface  to configure or via REST api if you prefer.

https://github.com/toni-moreno/snmpcollector/wiki/Component:-SNMP-Devices

Feel free to test and give me feedback about your experience.

**Comment by rpv-tomsk** [comment]
>It would be really appreciated if someone could implement a configurable timeout for snmp plugin.

That was added in 4300502aab2fa2ddf683d24e12b0e61f93ee4c5b and included in Collectd version 5.8.0.
